arangodb-net-standard/DocumentApi/PatchDocument.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace ArangoDBNetStandard.DocumentApi
+{
+    public class PatchDocument : DocumentBase
```

Comment:
I'd expect this model to be similar to the PostDocumentResponse and include the `New` and `Old` properties. According to documentation these should be populated if the `returnOld` or `returnNew` query parameters are used.